### ISSUE TITLE
TF 2.0 Documents how BatchNormalization update is handle in TF 2.0

### ISSUE THREAD

**Issue by jrabary** [tensorflow/tensorflow#25387]
When training model that need specific update such as batch norm mean and variance, it is not clear 
from the current version of documentations how we can handle this. 

This docs https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/Model suggests that we have to call `model.updates`

While here https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/generative/dcgan.ipynb there's nothing about `model.updates`

**System information**
- TensorFlow version: 2.0-preview

**Comment by lamberta** [comment]
model.updates is not in TF2 now. Thanks
